dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/configuration/Configuration.kt

```diff
@@ -259,6 +260,16 @@ internal constructor(
             return this
         }
 
+        /**
+         * Sets the strategy to schedule data uploads.
+         * @param uploadSchedulerStrategy the upload scheduler strategy,
+         * or null to use the default strategy (default: null)
+         */
+        fun setUploadScheduledStrategy(uploadSchedulerStrategy: UploadSchedulerStrategy?): Builder {
```

Comment:
Should it be called `setUploadSchedulerStrategy`?

```suggestion
        fun setUploadSchedulerStrategy(uploadSchedulerStrategy: UploadSchedulerStrategy?): Builder {
```